packages/theming/default-theme/src/defaultTheme.ts

```diff
@@ -40,6 +40,20 @@ function _defaultTypography(): Typography {
       heroSemibold: { face: 'primary', size: 'hero', weight: 'semiBold' },
       heroLargeStandard: { face: 'primary', size: 'heroLarge', weight: 'regular' },
       heroLargeSemibold: { face: 'primary', size: 'heroLarge', weight: 'semiBold' },
+      //v1 text variants
+      caption1: { face: globalTokens.font.family['base'] as FontFamily, size: globalTokens.font.size['100'] as FontSize, weight: globalTokens.font.weight.regular as FontWeightValue },
```

Comment:
In the future we should try to pull these from alias tokens so that these can change per platform without us needing to check for the current platform.

Reply:
And then we could collapse this and the win32 variants code for the FluentUI v2 variants into one